content/src/fieldConfig/profile.json

```diff
@@ -229,10 +248,32 @@
             "sole-proprietorship": "In a sole proprietorship you alone own the company and are responsible for its assets and liabilities. [Learn more about Sole Proprietorships.](https://business.nj.gov/pages/sole-proprietorship) "
           }
         },
-        "overrides": {
-          "OWNING": {
-            "description": "What is your `business structure|business-structure-learn-more`?"
+        "onboarding": {
+          "default": {
+            "header": "Which Business Structure Will You Choose?",
+            "headerContextualInfo": "",
+            "description": "Your business’ `business structure|business-structure-learn-more` can have a large impact on taxes, financing, liability, and closure of your business. We strongly suggest doing research on the best business structure for your business.  \nSelect a business structure for your Navigator profile below. Making a selection will not form your business, it will help us create your personalized guide. You can change your intended business structure anytime before formally filing it with the NJ Department of Treasury."
           },
+          "overrides": {
+            "OWNING": {
+              "description": "What is your `business structure|business-structure-learn-more`?"
+            },
+            "FOREIGN": {
+              "header": "Business Structure of Your Existing Out-of-State Business",
+              "description": "Please select the business structure used when you registered your business in your home state. You can find your business structure on your original `Certificate of Formation|certificate-formation`.",
+              "optionContent": {
+                "general-partnership": "",
+                "c-corporation": "",
+                "limited-liability-company": "",
+                "limited-liability-partnership": "",
+                "limited-partnership": "",
+                "s-corporation": "",
+                "sole-proprietorship": ""
+              }
+            }
+          }
+        },
+        "overrides": {
```

Comment:
so I can see this working now, in that the Dakota profile does not have the contextual link on the header. But I'm looking at the code and logic here, and I'm confused about _why_ it's working, can you explain the logic I'm missing?

It seems like for Dakota profile, it would take the default and then the overrides.

The default includes the `headerContextualInfo` field and the overrides for FOREIGN doesn't have headerContextualInfo set to empty string. So why doesn't the contextual info show up on the header for Dakota in profile?

Reply:
Like, if default is
```
"header": "Business Structure",
"headerContextualInfo": "business-structure-learn-more",
"description": "",
```

and the override is 
```
"overrides": {
  "FOREIGN": {
    "header": "Business Structure of Your Existing Out-of-State Business",
    "description": "Please select...",
```

Then wouldn't the merge result in an object like:
```
"header": "Business Structure of Your Existing Out-of-State Business",
"headerContextualInfo": "business-structure-learn-more",
"description": "Please select...",
```

What am I missing?

Reply:
So a few things. First I think it actually does work exactly how you described and your cache is invalid or something (like if you try this again and refresh the page you will probably get the correct behavior). 

This is my local Dakota Profile: 
<img width="1173" alt="image" src="https://user-images.githubusercontent.com/22485301/229893199-bb07340b-1f8a-4289-a3d5-5e4a5e30dfe5.png">




Whereas the design for Dakota Profile looks like this 

<img width="387" alt="image" src="https://user-images.githubusercontent.com/22485301/229893418-43dd132a-b496-424b-aae6-5b838f30cd4f.png">


The issue is, that some of the copy and design that I have for this is wrong (or at least I've questioned something and they've been amended). 

And I don't think the intent of this ticket was for me to question the design and copy and play ping pong with design and content and fix it all to be same as design which is what it kind feels like its turning into. 

So I would like to ask design and content to write a ticket for me once they've aligned and bring up my various questions during a meeting they have together.